### PR TITLE
Doc 9284 fix subdocument code

### PR DESCRIPTION
**Pull request by osfameron**
Run with `dotnet script modules/howtos/examples/SubDocument.csx` against a database with `travel-sample` installed.

### CODE REVIEW

File: modules/howtos/examples/SubDocument.csx [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+{
+Console.WriteLine("get:");
+// #tag::get[]
+    var result = await _collection.LookupInAsync("customer123", specs =>
+        specs.Get("addresses.delivery.country")
+    );
+
+    string country = result.ContentAs<string>(0);
+    WriteLine(country);
+// #end::get[]
```
Comment (maria-robobug) [inline modules/howtos/examples/SubDocument.csx]: Missing indentation here for this block. There's a few more below.

**Review by maria-robobug — APPROVED** [review]
👍 LGTM, just minor comment. 
Also we'll need to remember to update all `.csx` files when there's an SDK release to make sure we keep these up to date. 

Maybe we can create a script to update the `#r "nuget: CouchbaseNetClient, 3.2.0"` dependency for all `.csx` files or something.

**Comment by osfameron** [comment]
> Maybe we can create a script to update the `#r "nuget: CouchbaseNetClient, 3.2.0"` dependency for all `.csx` files or something.

the docs suggest `NuGet.Config` is the way to go for this. I've not played with it but will ticket it... https://issues.couchbase.com/browse/DOC-9453

File: modules/howtos/examples/SubDocument.csx (hunk 1 above)
Comment (osfameron) [reply modules/howtos/examples/SubDocument.csx]: Hmm, I think this was deliberate. 
I can kinda see the reason, but now disagree with it... so fixing as you  suggest 😂

**Pull request merged**